packages/button/src/Button.tsx

```diff
@@ -333,6 +352,13 @@ const Button = React.forwardRef((props: ButtonProps, forwardRef) => {
     'children',
   ]);
 
+  const modifiedGlyph =
```

Comment:
Thinking that maybe when we modify the glyph we should also change the size of the Icon based on the size of the button. What are your thoughts on this @DesignerDave?

Reply:
I think let's stick with the mocks on this one. We aren't changing the font size either I believe.

Reply:
When we do the flat button upgrade, we can consider making larger changes.

Reply:
Kk sounds good to me